apps/quick-learn-backend/src/routes/users/dto/list-filter.dto.ts

```diff
@@ -1,11 +1,16 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { UserTypeCodeEnum } from '@quick-learn/shared';
-import { IsEnum, IsOptional } from 'class-validator';
+import { IsBoolean, IsEnum, IsOptional } from 'class-validator';
 
 // DTO for the consult listing filter
 export class ListFilterDto {
   @ApiProperty({ example: 'admin', required: false })
   @IsOptional()
   @IsEnum(UserTypeCodeEnum)
   user_type_code?: string;
+
```

Comment:
As I have seen, this will not be needed as the API controller has a hard-coded value.